Makefile

```diff
@@ -71,3 +71,17 @@ else
 endif
 
 
+test:
+	@go test ./application/casp
```

Comment:
Why not just ./... ?

Reply:
It seemed all test scripts starts running in parallel which makes test cases fail

Reply:
I see. Try add `-p 1 -parallel 1` flags to go test